admin/sql/updates/20190422-mbs-9973-collection-added-date.sql

```diff
@@ -0,0 +1,17 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+ALTER TABLE editor_collection_area ADD COLUMN added TIMESTAMP WITH TIMESTAMP NOT NULL DEFAULT NOW();
```

Comment:
These have NOT NULL but the definitions in admin/sql/CreateTables.sql don't. I think we should allow NULL because we have no other value for existing items.

Reply:
Oops, yet another glitch, fixed in commit https://github.com/metabrainz/musicbrainz-server/pull/1046/commits/5a55f6d2c2fd92e87aa39b0ef5f78f02803be6e7.